packages/lit-dev-content/site/tutorials/content/first-component/03.md

```diff
@@ -0,0 +1,66 @@
+By the end of this step, you'll render one single word from the `words`
+property, based on some internal reactive state in our component.
+
+[_Internal reactive
+state_](/docs/components/properties/#internal-reactive-state) refers to reactive
+properties that are not part of the component's public API, therefore don't have
+an associated attribute.
+
+Define `idx`, an internal state property that will track which word is shown.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+class WordViewer extends LitElement {
+  @state() idx = 0;
```

Comment:
Solution and following step code includes `private` here.
```suggestion
  @state() private idx = 0;
```

Also, the `words` property has type annotation but this doesn't. I tend to drop them when it can be inferred from initial assignment but maybe it would be good to be consistent?